test/OverlayTriggerSpec.js

```diff
@@ -111,6 +111,33 @@ describe('<OverlayTrigger>', () => {
     wrapper.assertSingle('div.test');
   });
 
+  it('Should show after mouseover trigger', (done) => {
+    const clock = sinon.useFakeTimers();
+
+    try {
+      const wrapper = mount(
+        <OverlayTrigger overlay={<Div className="test" />}>
```

Comment:
did not specify the trigger prop so that it uses the defaults (the other tests in this file use the trigger="click") so we can test the mouseover / mouseout function